### ISSUE TITLE
[enh] Standardize parameter names

### ISSUE THREAD

**Issue by helenea** [openmole/openmole#446]
Suggestion: have the same parameter name in NSGA2Evolution and ProfileEvolution for the objective values. Currently, it is respectively "objectives" and "objective".

(yes it took me way too much time to notice that the error I was getting was because of the plural :sob:)

**Comment by JusteRaimbault** [comment]
Yes because NSGA2 is multi-objective whereas the profile method can not have more than one objective, so the names are standardized in the sense that they describe the methods :octocat:

**Comment by helenea** [comment]
That's true. However, since I often do a calibration first and then a profile, I usually just copy-paste the parameters from the calibration to the profile function, since there are so many in common. So I still think it's error inducing to have names that vary just by the singular/plural form.
I think it would be easier to debug an error on the number of objectives you pass to ProfileEvolution rather than spotting the "s" in the parameter name. But maybe it's just me ^^

**Comment by romainreuillon** [comment]
I really wouldn't mind setting everything to it's singular form, if there is an agreement on this.

**Comment by JusteRaimbault** [comment]
Indeed a solution would be to use the singular and consider that the objective is the vector of objectives in nsga2. Would agree with this.
(and then adding an explicit check on number of objectives in profile would be also useful if it is not already done)

**Comment by helenea** [comment]
The check on number of objectives in profile already exists!
(I made this error too and it was very easy to understand and correct, contrary to the vicious "s" :sweat_smile: )
